.github/workflows/ci_tests_backward.yaml

```diff
@@ -0,0 +1,105 @@
+# This workflow installs PyGMT and runs tests
+
+name: GMT Backward Tests
+
+on:
+  # push:
+  #   branches: [ main ]
+  pull_request:
+    # types: [ready_for_review]
+    paths-ignore:
+      - 'doc/**'
+      - 'examples/**'
+      - '*.md'
+      - 'README.rst'
+      - 'LICENSE.txt'
+      - '.gitignore'
+  # Schedule tests on Monday/Wednesday/Friday
+  schedule:
+    - cron: '0 0 * * 1,3,5'
```

Comment:
Do we need to test so frequently for GMT 6.3? Maybe just Tuesday and Thursday?

Reply:
I'm OK with a less frequent CI job for old GMT versions.

At line 9, perhaps we also don't need to run the CI job for when a PR is marked as ready for review.

Reply:
> Do we need to test so frequently for GMT 6.3? Maybe just Tuesday and Thursday?

Updated in 8e729ebb950506d00ac50c4b660ddb0a71d8e911.